lib/network_utils.pm

```diff
@@ -17,6 +17,7 @@ use Exporter;
 use strict;
 use warnings;
 use testapi;
+use utils qw(systemctl);
```

Comment:
Please use single-ticks for single word imports, i.e. `use utils 'systemctl'`. Actually, perlcritic should complain about this